src/drivers/PortFolder/WDMTestingTemplate/driver/fail_driver1.c

```diff
@@ -53,8 +65,10 @@ DriverEntry(
     DriverObject->MajorFunction[IRP_MJ_POWER]            = (PDRIVER_DISPATCH)DispatchPower;
     DriverObject->MajorFunction[IRP_MJ_SYSTEM_CONTROL]   = DispatchSystemControl;
     DriverObject->MajorFunction[IRP_MJ_PNP]              = (PDRIVER_DISPATCH)DispatchPnp;

```

Comment:
Does this driver compile if SET_DISPATCH is set to 0? I expect it would fail because there would be no declaration of DispatchPnp and DispatchCreate, right?

Reply:
Yeah, it won't compile. That's why I set SET_DISPATCH to 1 in all test files other than those (DispatchMismatch and DispatchAnnotationMissing) queries. For those I will set it to 0 as the test will depend on missing declarations for DispatchPnp and DispatchCreate.

Reply:
OK.  As this template evolves we'll want to document somewhere the various defines that are expected or necessary to get a test case to compile and run.